Form-Controls/styles.css

```diff
@@ -0,0 +1,288 @@
+
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@1,300&display=swap');
+
+/* root */
+
+:root {
+    --blue: #1cc5e7;
+    --purple: #b49df8;
+    --pink: #f17ed2;
+    --blue-green: #15c395;
+    --gray: #aea3a6;
+    --green: #73f777;
+    --yellow: #f5f885; 
+    --aqua: #1cc5e7;
+
+}
+
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing:border-box;
+}
+
+body {
+    color: black;
+    font-family: ''Montserrat', sans-serif';
+    font-weight: 600;
+}
+
+/* header */
+
+header {
+    background-color: var(--purple);
+
+}
+h1 {
+    display: flex;
+    justify-content: center;
+    padding: 1rem;
+    font-family: 'Roboto';
+    color:aqua;
+
+}
+main {
+    background-color: var(--yellow);
+    padding: 1rem;
+}
+
+/* form */
+
+form{
+    display: grid;
+    grid-template-columns:repeat(1 , fr) ;
```

Comment:
Are these styles being picked up? Have a look at the styles panel in chrome devtools